tests/support.py

```diff
@@ -211,6 +221,12 @@ def cleanpath(relative_path, test_case):
     return tmp_path
 
 
+def fixturepath(relative_path):
+    tmp_path = os.path.join(TEST_DATA_DIR, relative_path)
+    assert (not stat.ISDIR(os.stat(tmp_path)))
```

Comment:
This line is invalid AFAICT and would throw an exception if `fixturepath` were in fact used. I suppose it should be `stat.S_ISDIR(...)` as suggested when called interactively:
```
0|~/ku/migrid-trunk > python3 -c 'import stat; stat.ISDIR'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: module 'stat' has no attribute 'ISDIR'. Did you mean: 'S_ISDIR'?
```

Reply:
Apologies for this, it must have crept in because whenever I deal with asserts I check the failure conditions quite carefully and if its test assertions then failures are the 'input' I use when refining assertion messages.